src/api/app/views/webui2/webui/project/_new_package_branch_modal.html.haml

```diff
@@ -0,0 +1,50 @@
+.modal.fade#new-package-branch-modal{ tabindex: '-1', role: 'dialog', aria: { labelledby: 'new-package-branch-modal-label', hidden: true } }
+  .modal-dialog.modal-lg{ role: 'document' }
+    .modal-content
+      .modal-header
+        %h5.modal-title#new-package-branch-modal-label
+          Add new package branch to #{project.name}
+        %button.close{ type: 'button', data: { dismiss: 'modal' }, aria: { label: 'close' } }
+          %span{ aria: { hidden: true } }
+            &times
+      .modal-body.ui-front
+        %p
+          By branching a package from another project you add the package and its files
+          to your project in a transparent way. Changes of the files in the original
+          project will also change the files in the branched package in your project.
+        %p
+          You can add patches to the branched package without affecting the original
+          package.
+        - unless remote_projects.empty?
+          %p
+            Branches also work across interconnected OBS instances. This OBS has
+            #{pluralize(remote_projects.count, 'interconnect.', plural: 'interconnects.')}
+          %ul.list-inline
+            - @remote_projects.each do |_id, name, title|
+              %li.list-inline-item
+                = link_to("#{name}: #{title}", project_show_path(project: name))
+        = form_tag(package_branch_path) do
+          .form-group#orginal-project-name
+            %label{ for: 'linked_project' } Orginal project name
+            .input-group
+              .input-group-prepend
+                %span.input-group-text#search-icon-project
+                  %i.fa.fa-search
+                  %i.fas.fa-spinner.fa-spin.d-none
+              = text_field_tag 'linked_project', nil, size: 80, class: 'form-control', placeholder: 'Type to autocomplete', required: true
+          .form-group#original-package-name
+            %label{ for: 'linked_package' } Orginal package name
+            .input-group
+              .input-group-prepend
+                %span.input-group-text#search-icon-package
+                  %i.fa.fa-search
+                  %i.fas.fa-spinner.fa-spin.d-none
+              = text_field_tag 'linked_package', nil, size: 80, class: 'form-control', placeholder: 'Type to autocomplete', required: true
+          = hidden_field_tag 'target_project', project
+          = render partial: 'shared/package_branch_form', locals: { show_project_field: false, target_project: project,
+                                                                    package: nil, revision: nil }
```

Comment:
In the old ```package_branch_form``` the used the ```@revision``` and ```@package``` instance variable, but i think in this scope it's always ```nil``` right? So i didn't passed those through the locals.